stories/Section.stories.mdx

```diff
@@ -0,0 +1,95 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+import { linkTo } from '@storybook/addon-links';
+import { Section, Text } from '../src'; // components you used in the story
+
+import {
+    StandardSection,
+    RowSection,
+    StylingSection,
+    NavigationSection,
+} from './examples/Section';
+
+<Meta title="Interfacing/Section" component={Section} />
+
+# Section
+
+The Section component works like a flexbox grid container with a fixed `width` and `height`. It can be used to create a layout for your components.
+It is one of the most useful component to have. It is highly customizable thanks to dozens of props.
+
+## Table of Contents
+
+In this documentation we talk about the Section component, it's functionality and props.
+
+- [Functionality](#functionality)
+    - [Row Section](#row-section)
+    - [Styling Section](#styling-section)
+    - [Navigation Section](#navigation-section)
+- [Other Props & Functionalities](#other-props--functionalities)
+
+## Functionality
+
+The Section component can handle from 2 to 12 slots where content can be added and arranged into a row or grid.
```

Comment:
Where is this "slot" information coming from? Also refrain from using grid as stated above. This component is a flexbox used to help with creating layout using flex functionalities